docs/blueprints/EP031.rst

```diff
@@ -249,7 +249,8 @@ V. Events
 ==========
 
   1. Listening
```

Comment:
Right, indeed, initially I was assuming that it'd behave like a down UNI, since I hoping that eventually it'd probably be going up again since it would still exist: 

> If a proxy port source or destination status changes to DOWN or DISABLED (or if the loop stoops being a loop) all of associated EVCs should be considered not active, similar to what has been implemented when a UNI changes its UP state. No flows should be removed. Currently, mef_eline doesn't allow an EVC to be deactivated, so this will need to be published via events, depending if an EVC also starts to use status and status_reason then interruptions can also be used.

But, indeed, it could actually just remove the flows and allow the traffic to fallback to `mef_eline`. It's to you @italovalcy and @jab1982 to decide the behavior you'd like to see here, flow removals and then eventually trying to deploy it again once it goes up or consistency results in positive might be the the smoothest option, what do you think?

Reply:
If we were to support this, we'd also need to keep track of the looped links and which EVCs are going over it and react accordingly, we'll need to keep track of proxy ports anyway so that wouldn't be too far away. Good thing that we do have the link timer up to also avoid too much unnecessary IO with FlowMods. When it goes up it could redeploy again. I'll wait for your confirmation, but indeed looks like falling back would be the optimal (and less surprising) approach.

Reply:
Final verdict: proxy port link down will remove INT flows, and when proxy port link goes up if the EVC is still up then it'll deploy again. I've updated text on https://github.com/kytos-ng/kytos/pull/402/commits/a699943ca8326582898d230d4145db2d0f1b7849